src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -323,19 +332,35 @@ def get_class_obj_and_candidates(library_name, class_name, importable_classes, p
 
 
 def _get_pipeline_class(
-    class_obj, config, load_connected_pipeline=False, custom_pipeline=None, cache_dir=None, revision=None
+    class_obj,
+    config,
+    load_connected_pipeline=False,
+    custom_pipeline=None,
+    hub_repo_id=None,
+    hub_revision=None,
+    class_name=None,
+    cache_dir=None,
+    revision=None,
```

Comment:
I suggest we keep these as `**kwargs`. Instead of `hub_repo_id` or `hub_revision`, we could also opt for `repo_id` or `revision`, respectively following what we do across the library for Hub related utilities.

Reply:
We need to keep `hub_revision` as the other `revision` argument is already used for Git revisions